deploy/template.yaml

```diff
@@ -1527,6 +1527,55 @@ Resources:
             Period: !FindInMap [ EnvironmentConfiguration, !Ref AWS::AccountId, tg500ErrorWindow ]
             Stat: Sum
 
+  FrontRestApiGateway5xxErrorsPercentage:
```

Comment:
There appears to be some duplication with the above alarms, although they're informed by metrics from the load balancer and target groups. They use the SUM of 5XX metrics over a much longer window.
Without really knowing the importance of these alarms being in this configuration I've opted to add a new one rather than adjust. Up for debate.

Reply:
I think it would be good to assess across all of these holistically... I'd be keen to see a clear list of each alarm and what it's purpose is - either as comments here, or in a runbook somewhere.

At the moment it looks like we have:
- `FrontLoadBalancer5xxErrors` - ALB 5XX errors (>2 in 5m) - triggers PD
- `FrontTargetGroup5xxErrors` - Target group 5XX errors (>50 in 5m) - triggers PD
- `FrontTargetGroup5xxPercentErrors` - Target group 5XX errors (>5% in 1m) - used by the canary deployments
- `FE5XXErrorAlarm` - API GW 5XX errors (>10% in 1m) - triggers PD
- `FrontRestApiGateway5xxErrorsPercentage` - API GW 5XX errors (>80% in 1m) - what you've just added

There might be a good reason to have lots of different ones - e.g. I can see value in having both %-based and absolute thresholds, and it seems reasonable that the canary alarm is a bit more sensitive than the P1 incident alarm; but at the moment it's not obvious what the rationale is.